### PR TITLE
Adjusted vsize

### PR DESCRIPTION
**Pull request by mattmarcello**

### CODE REVIEW

File: lib/presentation/common/compose_base/shared/compose_tx.dart [hunk 1]
```diff
@@ -1,64 +1,66 @@
...
+// import 'package:horizon/domain/entities/fee_estimates.dart';
+// import 'package:horizon/domain/entities/fee_option.dart' as FeeOption;
+// import 'package:horizon/domain/entities/utxo.dart';
+// import 'package:horizon/domain/repositories/compose_repository.dart';
+// import 'package:horizon/domain/repositories/utxo_repository.dart';
+// import 'package:horizon/domain/services/transaction_service.dart';
+// import 'package:horizon/presentation/common/compose_base/bloc/compose_base_state.dart';
+// import 'package:horizon/presentation/common/compose_base/bloc/compose_base_event.dart';
+// import 'package:logger/logger.dart';
+//
```
Comment (dime-squanderer) [inline lib/presentation/common/compose_base/shared/compose_tx.dart:11]: do we not need this anymore?

File: lib/presentation/common/usecase/get_max_send_quantity.dart [hunk 2]
```diff
@@ -32,28 +32,33 @@ class GetMaxSendQuantity {
...
+    throw Exception("invariant: GetMaxSendQuantity not supported for `BTC`");
+
     // we take quantity to max, less 1000 sat in the
     // case of btc, for computing the transaction size
     // tom make sure we accont for fee / change
-
-    final quantity = sendAsset.quantity - 1000;
+    //
+    // final quantity = sendAsset.quantity - 1000;
+    // //
```
Comment (dime-squanderer) [inline lib/presentation/common/usecase/get_max_send_quantity.dart:43]: what about here?

File: lib/presentation/common/compose_base/shared/compose_tx.dart (hunk 1 above)
Comment (mattmarcello) [reply lib/presentation/common/compose_base/shared/compose_tx.dart:11]: for now, no.  just using the "ComposeTxUseCase" everywhere, with emitted states transparent in the bloc handlers.

File: lib/presentation/common/usecase/get_max_send_quantity.dart (hunk 2 above)
Comment (mattmarcello) [reply lib/presentation/common/usecase/get_max_send_quantity.dart:43]: it doesn't do much but i think we can leave it.

**Review by dime-squanderer — APPROVED** [review]

**Pull request merged**